build.wake

```diff
@@ -117,10 +116,9 @@ global def runRocketChipGenerator options =
       Some name = name, Nil
       None = Nil
     which "java", "-cp", classpath, main,
-    relTargetDir,
-    rootPackage, topModule,
-    rootPackage, configs,
-    baseFileName
+    "-td", relTargetDir,
+    "-T", topModule,
+    "-C", configs, Nil
```

Comment:
Can you make sure these are the full paths to the top module/configs?

Reply:
they are full paths